picard/util/pipe.py

```diff
@@ -158,6 +161,7 @@ def __generate_filename(self, app_name: str, app_version: str) -> str:
         else:
             self.__pipe_parent_dir = self.PIPE_UNIX_DIR
             if not self.__pipe_parent_dir:
+                log.debug(f"Used fallback pipe path: {self.PIPE_UNIX_FALLBACK_DIR}\nCouldn't use: {self.PIPE_UNIX_DIR}")
```

Comment:
Actually it would be preferable to not log 2 lines. `\n` isn't the line separator on all systems (one can use `os.sep` or the like for that). We usually try to log things on one line, it makes search/grep easier (though there are cases where multiple lines logging is useful).

Also no need to preformat strings, `log.*()` supports `%` notation and arguments.
We usually use `%r` for paths (to prevent any encoding issue, %s could lead to an error in some cases if the path cannot be converted to string for some reason).

```python
if self.PIPE_UNIX_DIR:
    self.__pipe_parent_dir = self.PIPE_UNIX_DIR
    log.debug("Using pipe path: %r", self.__pipe_parent_dir)
else:
    self.__pipe_parent_dir = self.PIPE_UNIX_FALLBACK_DIR
    log.debug("Using fallback pipe path: %r", self.__pipe_parent_dir)
```
    
Sorry for nitpicking, just trying to give insights about Picard coding style.

Reply:
Okay, no worries, consistency is important.

Tho when I think about it, I'll log a warning when we switch to fallback, because it's rather uncommon behavior.